src/common/SelectableOptionRow.js

```diff
@@ -0,0 +1,51 @@
+/* @flow strict-local */
```

Comment:
It would be good to include in the commit message here what you intend to use this component for, it's not clear to me from the name.